### PR TITLE
Add test for mxnet python interface

### PR DESCRIPTION
**Pull request by mrodozov**
#### PR description:
Adds unit test for the mxnet python interface
Copied from here https://mxnet.apache.org/get_started/validate_mxnet#python
The test needs this changes:
https://github.com/cms-sw/cmsdist/pull/5667

#### PR validation:

The test runs

### CODE REVIEW

File: PhysicsTools/MXNet/test/test_mxnet.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python
+import mxnet as mx
+import numpy
+a = mx.nd.ones((2, 3))
+b = a * 2 + 1
+b.asnumpy()
+numpy.array([[ 3.,  3.,  3.],
+       [ 3.,  3.,  3.]], dtype=numpy.float32)
```
Comment (smuzaffar) [inline PhysicsTools/MXNet/test/test_mxnet.py]: I do not understand these last 2 lines. I think just copy pasted the example. These lines are generated by `b.asnumpy()`

Comment (mrodozov) [reply PhysicsTools/MXNet/test/test_mxnet.py]: I didn't understand either until I finally read the text above it - it was a code so I took it as part of the example 😅

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/29243	0f39e07 
- unit test is somewhat simple, but should serve the purpose
- jenkins tests unit test log shows that the test passed OK

**Comment by silviodonato** [comment]
merge
@santocch this PR add a new simple unit test

**Comment by santocch** [comment]
+1

**Pull request merged**